packages/types/src/primitive/Extrinsic/SignerPayload.ts

```diff
@@ -27,6 +30,7 @@ export interface SignerPayloadType extends Codec {
 // We can ignore the properties, added via Struct.with
 const _Payload: Constructor<SignerPayloadType> = Struct.with({
   address: 'Address',
+  doughnut: Option.with(Doughnut),
```

Comment:
I suggest you use `'Option<Doughnut>'` which will allow user replace the default implementation